packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -491,6 +496,45 @@ class ExpansionTile extends StatefulWidget {
   /// than supplying a controller.
   final ExpansionTileController? controller;
 
+  /// Whether this list tile is part of a vertically dense list.
+  ///
+  /// If this property is null then its value is based on [ListTileTheme.dense].
+  ///
+  /// Dense list tiles default to a smaller height.
+  ///
+  /// It is not recommended to set [dense] to true when [ThemeData.useMaterial3] is true.
+  final bool? dense;
+
+  /// Defines how compact the list tile's layout will be.
+  ///
+  /// {@macro flutter.material.themedata.visualDensity}
+  ///
+  /// See also:
+  ///
+  ///  * [ThemeData.visualDensity], which specifies the [visualDensity] for all
+  ///    widgets within a [Theme].
+  final VisualDensity? visualDensity;
+
+  /// Defines the order of rendering the tile and its children
+  ///
+  /// If set to true then [children] will be on top of the tile otherwise the [children]
+  /// will be rendered beneath the tile
+  final bool reverse;
```

Comment:
I am a bit conservative that we should should go down the route of adding the flag to tweak ui like this. We may starting to be asked to add more and more flags to tweak ui in other specific ways. I would suggest to build your own expansion tile if the current design doesn't fit.

All the other new properties seem reasonable though

Reply:
That seems reasonable. So, I will remove this flag. I will also mention the issue in the description. Thanks for the review.